ignite/src/main/java/org/hibernate/ogm/datastore/ignite/transaction/impl/IgniteTransactionCoordinatorBuilder.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.ignite.transaction.impl;
+
+import org.hibernate.ConnectionAcquisitionMode;
+import org.hibernate.ConnectionReleaseMode;
+import org.hibernate.ogm.datastore.ignite.impl.IgniteDatastoreProvider;
+import org.hibernate.resource.transaction.TransactionCoordinator;
+import org.hibernate.resource.transaction.TransactionCoordinatorBuilder;
+import org.hibernate.resource.transaction.spi.TransactionCoordinatorOwner;
+
+/**
+ * @author Sergey Chernolyas &amp;sergey_chernolyas@gmail.com&amp;
+ */
+public class IgniteTransactionCoordinatorBuilder implements TransactionCoordinatorBuilder {
+	private final TransactionCoordinatorBuilder delegate;
+	private final IgniteDatastoreProvider datastoreProvider;
+
+	/**
+	 * @param delegate transaction delegate
+	 * @param datastoreProvider instance of IgniteDatastoreProvider
+	 */
+	public IgniteTransactionCoordinatorBuilder(
+			TransactionCoordinatorBuilder delegate, IgniteDatastoreProvider datastoreProvider) {
+		this.delegate = delegate;
+		this.datastoreProvider = datastoreProvider;
+	}
+
+	@Override
+	public TransactionCoordinator buildTransactionCoordinator(
+			TransactionCoordinatorOwner owner,
+			TransactionCoordinatorOptions options) {
+		TransactionCoordinator coordinator = delegate.buildTransactionCoordinator( owner, options );
+		if ( delegate.isJta() ) {
+			//return new IgniteJtaTransactionCoordinator( coordinator, datastoreProvider );
+			//@todo add currect code of Victor
+			throw new UnsupportedOperationException( "Not supported yet" );
```

Comment:
Do you remove support of JTA?

Reply:
Hi @Z-z-z-z !
No.  JTA supports by your code. And ... it is main question. Can I migrate your code to IgniteJtaTransctionCoordinator like in all other dialects?

Reply:
@schernolyas I think we agreed that before changing/applying any of this you need to provide a test case.